administrator/language/de-DE/plg_system_stats.ini

```
@@ -5,24 +5,24 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_STATS="System - Joomla!-Statistikerhebung"
-PLG_SYSTEM_STATS_BTN_NEVER_SEND="Niemals"
-PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Immer"
+PLG_SYSTEM_STATS_BTN_NEVER_SEND="Nein"
+PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Ja"
 PLG_SYSTEM_STATS_BTN_SEND_NOW="Einmalig"
 PLG_SYSTEM_STATS_INTERVAL_LABEL="Intervall (Stunden)"
 PLG_SYSTEM_STATS_LABEL_CMS_VERSION="CMS-Version"
 PLG_SYSTEM_STATS_LABEL_DB_TYPE="DB-Typ"
 PLG_SYSTEM_STATS_LABEL_DB_VERSION="DB-Version"
-PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Joomla! bittet um Erlaubnis einige Basisstatistiken zu erheben"
+PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Systemstatistiken an Joomla! übermitteln."
```

```suggestion
PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Systemstatistiken an Joomla! übermitteln"
```

Ist nen title oder da brauchen wir keinen Punkt am ende oder?